src/coreclr/tools/aot/Mono.Linker.Tests/TestCasesRunner/AssemblyChecker.cs

```diff
@@ -1117,21 +1205,23 @@ private void VerifyGenericParameters (IGenericParameterProvider src, IGenericPar
 			}
 		}
 
-		private void VerifyParameters (IMethodSignature src, IMethodSignature linked)
+		private void VerifyParameters (IMethodSignature src, TrimmedMethodEntity linked)
 		{
-			Assert.Equal (src.HasParameters, linked.HasParameters);
+			Assert.Equal (src.HasParameters, linked.Method.Signature.Length > 0);
 			if (src.HasParameters) {
 				for (int i = 0; i < src.Parameters.Count; ++i) {
 					var srcp = src.Parameters[i];
-					var lnkp = linked.Parameters[i];
+					//var lnkp = linked.Parameters[i];
 
+#if false
```

Comment:
`VerifyCustomAttributes` has not been ported to NativeAOT yet, so it doesn't work. There's lot of code in this file which is not even ifdefed out, but never called, so you can see some possible callsites for this method, but none are actually active.

There's LOT of things to do to get even close to the ILLink functionality in this area. And lot of things are either not possible, or unreasonably hard because NativeAOT doesn't store the information, or the way it's stored is very different and complex.